### ISSUE TITLE
Compiler error references an empty page about tuples

### ISSUE THREAD

**Issue by gurdiga** [elm/elm-lang.org#809]
Hey! 👋

I just got an error which references this page: https://elm-lang.org/0.19.0/tuples, which is empty. 😶

For more context, here is the error:
```
-- BAD TUPLE ------------------------------------------ src/RegistrationForm.elm

I only accept tuples with two or three items. This has too many:

42|>        ( FullName.fromString model.fullName
43|>        , Login.fromString model.login
44|>        , Photo.fromFile <| UploadButton.file model.photo
45|>        , PhoneNumber.fromString model.phoneNumber
46|>        )

I recommend switching to records. Each item will be named, and you can use the
`point.x` syntax to access them.

Note: Read <https://elm-lang.org/0.19.0/tuples> for more comprehensive advice on
working with large chunks of data in Elm.
```

Wandering around in this repo, I’ve found the https://github.com/elm/elm-lang.org/blob/master/src/pages/0.19.0/tuple.elm page which seems to fit the error. 🤓 

Should the `/src/pages/0.19.0/tuple.elm` me moved to `/src/pages/0.19.0/tuples.elm`? 🤔

**Comment by JDemler** [comment]
There is already a PR which fixes this issue: #805

**Comment by gurdiga** [comment]
👍
